BlackPearl.Controls.Library/Control/MultiSelectCombobox.xaml.cs

```diff
@@ -122,6 +122,16 @@ private ObservableCollection<object> SuggestionItemsSource
             set => SetValue(SuggestionItemsSourceProperty, value);
         }
 
+        /// <summary>
+        /// Internal property - Selected index in Suggestion drop down
+        /// </summary>
+        private static readonly DependencyProperty SuggestionIndexProperty =
+            DependencyProperty.Register(nameof(SuggestionIndex), typeof(int), typeof(MultiSelectCombobox));
+        private int SuggestionIndex
+        {
+            get => (int)GetValue(SuggestionIndexProperty);
+            set => SetValue(SuggestionIndexProperty, value);
+        }
```

Comment:
Not required anymore. Please update your branch with latest master branch. Or just delete this.